server/db/schema.ts

```diff
@@ -113,6 +113,17 @@ export const verificationToken = pgTable(
   }),
 );
 
+export const emailVerificationToken = pgTable("EmailVerificationToken", {
+  id: serial("id").primaryKey(),
+  token: text("token").unique().notNull(),
+  createdAt: timestamp("createdAt").defaultNow().notNull(),
+  expiresAt: timestamp("expiresAt").notNull(),
+  email: text("email").notNull(),
```

Comment:
I think we will also want `.unique()` on the email, or else someone could take an existing account's email. We will want to make sure the email isn't in use as suggested by coderabbit AI too.

Reply:
True

Reply:
Done ✅